<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>whatToEnrichtDialog</class>
 <widget class="QDialog" name="whatToEnrichtDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject#whatToEnrichtDialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0795455 rgba(0, 57, 70, 255), stop:0.5625 rgba(0, 85, 104, 255), stop:0.954545 rgba(0, 107, 131, 255));}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>861</width>
       <height>471</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>861</width>
       <height>471</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QObject#frame{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0113636 rgba(113, 181, 190, 255));
	
border-radius:20px;}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>121</y>
        <width>841</width>
        <height>341</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>841</width>
        <height>341</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QListWidget" name="searchResult">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="searchResultLabel">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 81, 135);</string>
         </property>
         <property name="text">
          <string>Search Results</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>90</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Search Properties for Enrichment</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <widget class="QPushButton" name="searchButton2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>153</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>153</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Search Area Concept</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>40</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>153</width>
         <height>22</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="costumpropertyLabel">
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 81, 135);</string>
          </property>
          <property name="text">
           <string>In Area:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inAreaEditText">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="searchConceptButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>153</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>153</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Search Concept</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>541</width>
        <height>75</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>488</width>
        <height>75</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="conceptSearchLabel">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 81, 135);</string>
           </property>
           <property name="text">
            <string>Search ID Concept:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="conceptSearchEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>311</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>311</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>124</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="tripleStoreLabel">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 81, 135);</string>
           </property>
           <property name="text">
            <string>Triple Store:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tripleStoreEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>311</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>311</width>
             <height>22</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="applyButton">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>90</y>
        <width>153</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>153</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
